### PR TITLE
feat: Remove default exports

### PR DESCRIPTION
**Pull request by ffflorian**

### CODE REVIEW

File: packages/tslint-config/tslint.json [hunk 1]
```diff
@@ -15,6 +15,7 @@
     "jsx-boolean-value": false,
     "jsx-no-lambda": false,
     "jsx-no-multiline-js": false,
+    "no-default-export": true,
```
Comment (lipis) [inline packages/tslint-config/tslint.json]: 👍

Comment (lipis) [reply packages/tslint-config/tslint.json]: This will give bunch of errors in other projects.. but that's a good thing!

Comment (ffflorian) [reply packages/tslint-config/tslint.json]: Yes, I will mark it as breaking

**Comment by bennyn** [comment]
**Note:** I restored the `feat/no-default` branch because tests were running without errors on Chrome 73 and I wanted to check how it looks like with Chrome 74. Unfortunately our Store Engine tests for the IndexedDBEngine fail with Chrome 74. 😢

**Pull request merged**